### PR TITLE
Fix: python sdk archive project, correct application variables.

### PR DESCRIPTION
**Pull request by jmelinav**
**What this PR does / why we need it**:
- Variable names are incorrect in application.yaml
-  Member variable used in archive project is incorrect

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:no

```release-note
NONE
```

### CODE REVIEW

File: serving/src/main/resources/application.yml [hunk 1]
```diff
@@ -32,7 +32,8 @@ feast:
       enabled: false
       provider: http
       options:
-        basePath: http://localhost:3000
+        authorizationUrl: http://localhost:8082
+        subjectClaim: email
```
Comment (khorshuheng) [inline serving/src/main/resources/application.yml:36]: subjectClaim is part of the authentication options, rather than authorization.

**Comment by woop** [comment]
@jmelinav is this still open?

**Comment by jmelinav** [comment]
> @jmelinav is this still open?
Haven't tested this on the latest, I will close it and reopen it if necessary.

**Pull request closed without merging**